src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -19,6 +19,8 @@ public class CliSyntax {
     public static final Prefix PREFIX_EXAM_DESCRIPTION = new Prefix("ex/");
     public static final Prefix PREFIX_EXAM_DATE = new Prefix("ed/");
     public static final Prefix PREFIX_DEADLINE = new Prefix("dl/");
+    public static final Prefix PREFIX_MOD_NAME = new Prefix("mn/");
```

Comment:
Maybe can just shorten to m for PREFIX_MOD_NAME since prefixes can be reused for different commands so it doesn't need to be mn

Reply:
okay, would change this